src/libraries/System.Transactions.Local/src/Resources/Strings.resx

```diff
@@ -421,6 +421,6 @@
     <value>[Base]</value>
   </data>
   <data name="DistributedNotSupportOn32Bits" xml:space="preserve">
-    <value>Distributed transactions are currently unsupported on 32-bit version of the .NET runtime.</value>
+    <value>Distributed transactions are currently unsupported on the 32-bit version of the .NET runtime.</value>
```

Comment:
Maybe "are currently unsupported in 32-bit processes"?  Doesn't matter though if this is going to be fixed soon.

Reply:
Done the change, one never knows how long is soon:tm: